docs/source/new_project.rst

```diff
@@ -41,7 +41,7 @@ questions:
    * - *Would you like to include mypy to perform static type checking for type hints?*
      - `mypy <https://www.mypy-lang.org>`_ performs static type checking on python code that uses `type hints <https://docs.python.org/3/library/typing.html>`_. This type checking makes sure that the correct data types are being used where type hints are defined. If basic or strict type checking is selected, a pre-commit hook and GitHub actions workflow that perform the type checking are added. Basic type checking performs type checks but ignores code or imports for which type hints are not written. Strict type checking enforces type hints are used by giving errors where no type hints are found.
    * - *Do you want to create some example module code?*
-     - If this option is selected the template will create a model in ``src/{{module_name}}`` and create a corresponding example test file. Defaults to ``True`` during simple installation.
+     - If this option is selected the template will create a module in ``src/{{package_name}}`` and create a corresponding example test file. Defaults to ``True`` during simple installation.
```

Comment:
Not sure if this shouldn't be too long, but could we include the name of the module in the example path here? i.e. `src/{{package_name}}/example_module.py` (I think that's the name of the example module we use)

Reply:
Updated